integration_tests/test_checkov_json_report.py

```diff
@@ -58,6 +59,11 @@ def validate_json_quiet(self):
             self.assertFalse(data["results"]["passed_checks"])
             self.assertTrue(data["summary"])
 
+    def validate_check_in_report(self, report_path, check_id):
+        with open(report_path) as json_file:
+            data = json.load(json_file)[0]
+        assert any(check["check_id"] == check_id for check in data["results"]["failed_checks"] + data["results"]["passed_checks"])
```

Comment:
Using `itertools.chain` here results in not creating a new list with failed and passed checks.
```suggestion
        assert any(check["check_id"] == check_id for check in itertools.chain(data["results"]["failed_checks"], data["results"]["passed_checks"]))
```
and don't forget to add the import 😉